tests/unit/scanners/test.filename.js

```diff
@@ -159,6 +159,8 @@ describe('Reserved file names', () => {
       const { linterMessages } = await filenameScanner.scan();
       expect(linterMessages.length).toEqual(1);
       expect(linterMessages[0].code).toEqual(messages.RESERVED_FILENAME.code);
+      expect(linterMessages[0].message).toEqual('Reserved filename found.');
+      expect(linterMessages[0].description.startsWith('Files whose names are reserved')).toEqual(true)
```

Comment:
These two lines could directly assert against `RESERVED_FILENAME.message` and `RESERVED_FILENAME.description` couldn't they?

Reply:
Then I actually wouldn't match/notice the `undefined` `description` property which is what I'm after here.

Reply:
Ah yes, fair enough 👍